### PR TITLE
Fix loader in showcase and add it to screenshots

### PR DESCRIPTION
**Pull request by sea-snake**
- Fix loader in showcase, make sure the astronaut image loads and doesn't animate.
- Add the loader screen to screenshots, including the "taking forever" variant that shows after 10 seconds.

<hr/><details><summary>🟢 Some screens were added</summary></details><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: src/frontend/src/components/loader.ts [hunk 1]
```diff
@@ -1,13 +1,20 @@
 import { ERROR_SUPPORT_URL } from "$src/config";
 import { html, render } from "lit-html";
-import loaderUrl from "./loader.svg";
+
+// Use same import approach as in 'src/frontend/src/flows/dappsExplorer/dapps.ts'
```
Comment (lmuntaner) [inline src/frontend/src/components/loader.ts:4]: Ah, nice, maybe we can do the same in the other and it fixes the issue with the types?

File: src/frontend/src/styles/main.css [hunk 2]
```diff
@@ -3149,6 +3149,7 @@ input[type="checkbox"] {
   animation-duration: 5s;
   animation-iteration-count: infinite;
   animation-timing-function: ease-in-out;
+  transform: translate(-50%, -50%);
```
Comment (lmuntaner) [inline src/frontend/src/styles/main.css:3152]: Why is this necessary?

File: src/frontend/src/components/loader.ts [hunk 3]
```diff
@@ -33,8 +40,11 @@ const startLoader = () => {
   };
 };
 
-export const withLoader = async <A>(action: () => Promise<A>): Promise<A> => {
-  const endLoader = startLoader();
+export const withLoader = async <A>(
+  action: () => Promise<A>,
+  showCheckOngoingIssues?: boolean
```
Comment (lmuntaner) [inline src/frontend/src/components/loader.ts:45]: Where do you pass this? I don't see it used in any showcase page nor anywhere else...

**Review by lmuntaner — COMMENTED** [review]
Thanks! A few questions but it's looking very good!

File: src/frontend/src/styles/main.css (hunk 2 above)
Comment (sea-snake) [reply src/frontend/src/styles/main.css:3152]: When I disable the animation in the showcase (for screenshots), the logo isn't centered anymore because `translate(-50%, -50%)` was part of the animation. With this change the logo is always centered even if the animation is disabled..

File: src/frontend/src/components/loader.ts (hunk 3 above)
Comment (sea-snake) [reply src/frontend/src/components/loader.ts:45]: It's used in the `loaderTakingForever.astro` file, to avoid having to wait the 10 seconds, there's two loading screen screenshots, one with button and one without.

File: src/frontend/src/styles/main.css (hunk 2 above)
Comment (lmuntaner) [reply src/frontend/src/styles/main.css:3152]: noted, thanks!

File: src/frontend/src/components/loader.ts (hunk 1 above)
Comment (sea-snake) [reply src/frontend/src/components/loader.ts:4]: This doesn't fix the issue with types, it fixes the issue with the actual imported data being inconsistent between showcase and build.

File: src/frontend/src/components/loader.ts (hunk 3 above)
Comment (lmuntaner) [reply src/frontend/src/components/loader.ts:45]: Oh, sorry. The `true` was on the other line in my screen and I missed it

**Review by lmuntaner — APPROVED** [review]

File: src/frontend/src/components/loader.ts (hunk 1 above)
Comment (lmuntaner) [reply src/frontend/src/components/loader.ts:4]: thanks for the explanation!
